RTD Mobile header brand adjustment

kyleaton opened the pull request:
Please annotate your PR title to describe what the PR does, then give a brief bulleted description of your PR below. PR titles should begin with [BUGFIX], [FEATURE], [DOCS], [MAINTENANCE], or [CONTRIB]. If a new feature introduces breaking changes for the Great Expectations API or configuration files, please also add [BREAKING]. You can read about the tags in our [contributor checklist](https://docs.greatexpectations.io/docs/contributing/contributing_checklist).

Changes proposed in this pull request:
- Brand updates to RTD Nav
-
-


After submitting your PR, CI checks will run and @cla-bot will check for your CLA signature.

For a PR with nontrivial changes, we review with both design-centric and code-centric lenses.

In a design review, we aim to ensure that the PR is consistent with our relationship to the open source community, with our software architecture and abstractions, and with our users' needs and expectations. That review often starts well before a PR, for example in GitHub issues or Slack, so please link to relevant conversations in notes below to help reviewers understand and approve your PR more quickly (e.g. `closes #123`).

Previous Design Review notes:
-
-
-


### Definition of Done
Please delete options that are not relevant.

- [ ] My code follows the Great Expectations [style guide](https://docs.greatexpectations.io/docs/contributing/style_guides/code_style)
- [ ] I have performed a [self-review](https://docs.greatexpectations.io/docs/contributing/contributing_checklist) of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added [unit tests](https://docs.greatexpectations.io/docs/contributing/contributing_test#writing-unit-and-integration-tests) where applicable and made sure that new and existing tests are passing.
- [ ] I have run any local integration tests and made sure that nothing is broken.


Thank you for submitting!

kyleaton committed:
add footer items

kyleaton committed:
adding different size logos

kyleaton committed:
adding in v2 docs

kyleaton committed:
add in new logo

kyleaton committed:
logo for rtd

kyleaton committed:
new font

kyleaton committed:
replace primart color in css

kyleaton committed:
replacing inline styles for primary colors

kyleaton committed:
rtd style updates

kyleaton committed:
docs nav fixes

kyleaton committed:
fix conflict

kyleaton committed:
merge conflicts

kyleaton committed:
rtd mobile header

kyleaton committed:
Merge branch 'develop' into docs-version-dropdown

ghost commented:
<details>
<summary><b>👇 <a href="https://app.codesee.io/r/reviews?pr=6557&src=https%3A%2F%2Fgithub.com%2Fgreat-expectations%2Fgreat_expectations">Click on the image</a> for a new way to code review</b>
</summary>

  - Make big changes easier — review code in small groups of related files
  - Know where to start — see the whole change at a glance
  - Take a code tour — explore the change with an interactive tour
  - Make comments and review — all fully sync’ed with github

    [Try it now!](https://app.codesee.io/r/reviews?pr=6557&src=https%3A%2F%2Fgithub.com%2Fgreat-expectations%2Fgreat_expectations)
</details>

[![Review these changes using an interactive CodeSee Map](https://s3.us-east-2.amazonaws.com/maps.codesee.io/images/github/great-expectations/great_expectations/6557/62da6a60/2605ce75d8f521f344db7f4972da8f646fd43e0d.svg)](https://app.codesee.io/r/reviews?pr=6557&src=https%3A%2F%2Fgithub.com%2Fgreat-expectations%2Fgreat_expectations)

#### Legend
![CodeSee Map Legend](https://app.codesee.io/Legend_v2.png)

kyleaton committed:
Merge branch 'develop' into docs-version-dropdown

kyleaton committed:
Update custom.scss

kyleaton committed:
Merge branch 'develop' into docs-version-dropdown

kyleaton committed:
Merge branch 'develop' into docs-version-dropdown

kyleaton committed:
Update link to v2 to correct link

kyleaton committed:
styling for nav sub menu

kyleaton committed:
Merge branch 'develop' into docs-version-dropdown

kyleaton committed:
Merge branch 'develop' into docs-version-dropdown

anthonyburdi commented on docs_rtd/_templates/layout.html:
```diff
@@ -168,193 +168,80 @@
+          </li>
+          <li>
+            <a class="header-item" href="https://greatexpectations.io/case-studies"
+              >CAREERS</a
```
Should these go to different pages (i.e. not the case-studies page)? Or are those pages not yet ready?

anthonyburdi commented on docs_rtd/_templates/layout.html:
```diff
@@ -168,193 +168,80 @@
-            </a>
+            <a class="header-item" href="https://greatexpectations.io/integrations"
+              >INTEGRATIONS</a
+            >
```
Is integrations duplicated?

anthonyburdi reviewed (commented):
Just have a few questions.

kyleaton committed:
Addressing comments and fixing links

kyleaton commented:
@anthonyburdi addressed comments with [b5346fc](https://github.com/great-expectations/great_expectations/pull/6557/commits/b5346fc7219e91d31fc3568e9d5318b72fe09ba8)

Thanks!

kyleaton committed:
Merge branch 'develop' into docs-version-dropdown

anthonyburdi reviewed (approved):
LGTM!
